drivers/lds01_dma/include/lds01.h

```diff
@@ -0,0 +1,121 @@
+/*
+ * Copyright (C) 2021 COGIP Robotics association <cogip35@gmail.com>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup    driver_lds01 LDS01 LiDAR driver
```

Comment:
Should the doxygen names updated ?
Like for example driver_lds01 => drier_lds01_dma ?

Reply:
Done